build.gradle

```diff
@@ -52,7 +52,9 @@ sourceSets {
   cli {
       java {
           srcDir 'util/src'
-          compileClasspath += main.output + harness.output
+          srcDir 'javarosa/util/schema-gen/src'
```

Comment:
Gradle isn't being my friend these days and I can't figure out how to reference other sourcesets in a sourceset def. So I just copied them. This is needed to get the CLI building again